docs/source/user/transformations/aggregation-sum.ipynb

```diff
@@ -0,0 +1,419 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "bounds = (0., 10.)\n",
+    "sizes = list(range(1, 10_000_000, 10_000))\n",
+    "\n",
+    "pd.DataFrame({\n",
+    "    \"dataset size\": sizes,\n",
+    "    \"sequential\": [make_sized_bounded_float_checked_sum(size, bounds, S=\"Sequential<f64>\").map(0) for size in sizes],\n",
+    "    \"pairwise\": [make_sized_bounded_float_checked_sum(size, bounds, S=\"Pairwise<f64>\").map(0) for size in sizes],\n",
+    "}).plot(0, ylabel=\"sensitivity\");"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The increase to sensitivity when using a sequential summation algorithm grows on the order of $O(n^2)$, while the pairwise algorithm grows on the order of $O(log_2(n)n)$.\n",
+    "\n",
+    "OpenDP defaults to the pairwise algorithm, but the ability to configure the algorithm can be useful to calculate the sensitivity in situations where you don't have control over how the summation is computed.\n",
+    "For example, floating-point aggregations in SQLite and MySQL both exhibit increases in sensitivity akin to the sequential algorithm.\n",
+    "\n",
+    "Beware, these relaxation terms grow far more quickly when the data type is adjusted to single-precision floats (`f32`)!\n",
+    "\n",
+    "\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Specialized Integer Constructors\n",
+    "\n",
+    "Just as in the case for floating-point types, there are specialized constructors for summation over integral types.\n",
+    "\n",
+    "The integral measurements from `make_bounded_sum` and `make_sized_bounded_sum` each use properties of the bounds, data types and input metric to determine which strategy to use to compute the sum in a way that protects the sensitivity from the effects of numerical overflow.\n",
```

Comment:
Saying "measurement" here is a little confusing, is that what you wanted? Or maybe something like "operations" or "computations"?